compiler-rt/lib/mpk_untrusted/alloc_site_handler.h

```diff
@@ -42,6 +42,10 @@ class AllocSite {
   /// Returns a shared pointer to the Error AllocSite.
   static std::shared_ptr<AllocSite> error();
 
+  // Note : containsPtr contains potentially wrapping arithmetic. If a ptr
+  // and the allocations size exceed 0xffffffffffffffff, then any pointer
```

Comment:
This comment is a bit confusing. 

1. since a ptr happens to be 64 bits in our case, it just happens to correspond to  0xfff...
2. I don't see any change to the internal check, so I'm unsure of how this will always return false.